pkgs/by-name/ta/taterclient-ddnet/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  ddnet,
+  fetchFromGitHub,
+}:
+let
+  clientExecutable = "TaterClient-DDNet";
+in
+ddnet.overrideAttrs (
```

Comment:
It's usually discouraged to override packages like this within Nixpkgs, as this could lead to infinite recursions with an overlay like this:

```nix
final: prev: {
  ddnet = final.taterclient-ddnet;
}
```

Reply:
Tu as raison, c'est en fait préférable de l'écrire from scratch.